test/functional/testlib.bash

```diff
@@ -147,6 +147,16 @@ print_stack() {
 
```

Comment:
Spelling: frmework -> framework (in commit message)

Reply:
damn typos! haha, thanks